src/Service/PDFService.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Service;
+
+use App\Helper\Participant;
+use Mpdf\Mpdf;
+use Mpdf\MpdfException;
+use Mpdf\Output\Destination;
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Error\SyntaxError;
+
+class PDFService
+{
+    private $mpdf;
+    private Environment $twig;
+
+    public function __construct(Environment $twig)
+    {
+        $this->mpdf = new Mpdf(['orientation' => 'L', 'format' => [60.96, 101.6], 'margin_left' => 5, 'margin_right' => 5, 'margin_top' => 5, 'margin_bottom' => 2]);
+        $this->twig = $twig;
+    }
+
+    /**
+     * @throws MpdfException
+     * @throws SyntaxError
+     * @throws RuntimeError
+     * @throws LoaderError
+     */
+    private function renderPDF(string $name, string $orderID, \DateTime $DOB, string $specimenID, string $moduleNum, string $timePoint, string $sampleCode, string $VisitType, string $collectionVolume): void
+    {
+        $this->mpdf->WriteHTML(
+            $this->twig->render('program/nph/PDF/biospecimenLabel.html.twig', [
```

Comment:
To be consistent with other twig files naming convention in our project could you rename this file to biospecimen-label.html.twig? Also, the directory name to pdf?